mitxgraders/comparers/comparers.py

```diff
@@ -102,7 +103,7 @@ class EqualityComparer(Comparer):
 
     The following example takes the norm of the expected answer and student input
     before comparison. Note the different method of changing the comparer.
-    >>> FormulaGrader.set_default_comparer(EqualityComparer(transform=np.linalg.norm))
+    >>> MatrixGrader.set_default_comparer(EqualityComparer(transform=np.linalg.norm))
```

Comment:
Oops, my bad!

Reply:
This no longer matters since I just reverted MatrixGrader.default_comparer to `equality_comparer`, but I still think it's nicer to use class override :)